### ISSUE TITLE
core:update verbosity + changelog announcement

### ISSUE THREAD

**Issue by ThaDafinser** [piwik/piwik#8318]
I'm currently upgrading from 2.13.1 to 2.14

In the release announcement is nothing mentioned about a database upgrade, but there is one: https://github.com/piwik/piwik/blob/master/core/Updates/2.14.0-b2.php

I thought okay that's not so big. But the upgrade also executes following command:

`ALTER TABLE piwik_log_visit DROP COLUMN config_windowsmedia` which takes in my enviroment quiet a lot of time.

-> Okay that's caused by #8304 , but -vvv would be awesome still :+1: 

I think thats the 2nd time the database changes were not mentioned in the release notes, because they were introduced in a beta.

Is it possible to introduce the verbosity level to the core:update command? Because i already thought this time there is something broken, because it took so long (and there was only a drop + create table in the diff)

![image](https://cloud.githubusercontent.com/assets/533017/8613965/3d698054-26e5-11e5-859b-79d2ec308518.png)

**Comment by mattab** [comment]
Hi @ThaDafinser 

As you suspect we didn't announce the `site_settings` update since it's not major and should always be very fast.

> ALTER TABLE piwik_log_visit DROP COLUMN config_windowsmedia which takes in my enviroment quiet a lot of time.

This is surprising, because this query should not execute when upgrading to 2.14.0! we still have the column in the database which is handled by this class: https://github.com/piwik/piwik/blob/8290/plugins/DevicePlugins/Columns/PluginWindowsMedia.php

Do you have this file in your Piwik instance? Maybe it was deleted somehow and then Piwik deleted the column during upgrade?

I checked on local instance and on the demo and the column `config_windowsmedia ` is still there.

Do you maybe see a warning in Administration > System check?

**Comment by ThaDafinser** [comment]
@mattab i think my system should be fine. It seems many people had/have that issue...

Since you will lose data, if you not backup all...dont you want to "unpublish" the current release, until the error is found?

![image](https://cloud.githubusercontent.com/assets/533017/8618908/bc316d0c-2712-11e5-9fdb-ffc2d3f6a64a.png)
![image](https://cloud.githubusercontent.com/assets/533017/8618916/ce1f4020-2712-11e5-9357-51682061f36b.png)
![image](https://cloud.githubusercontent.com/assets/533017/8618918/d57cb74e-2712-11e5-8c97-b3c2e3549f22.png)

**Comment by mattab** [comment]
Does anyone else experienced this issue during 2.14.0 upgrade? please leave a comment. I'm still very surprised by this bug and don't understand yet how it could happen :-(

**Comment by ThaDafinser** [comment]
As a side note maybe: It dropped me more than that column. All columns mentioned here: http://forum.piwik.org/read.php?2,127838,page=1#msg-127877

I also used the workaround to "recreate" the columns, because i more likely lose that (unimportant) data then the whole tracking day.

**Comment by ChristianGfK** [comment]
We also experienced this after upgrading to 2.14.0.
Kind of scary to see this when looking at your stats: 
![bpmaood](https://cloud.githubusercontent.com/assets/6390522/8746100/66eb4998-2c86-11e5-9d2f-1a3a18fab8d4.png)

I added the columns mentioned in the forum post and visitor tracking resumed.
Is the data collected between upgrading to 2.14.0 (yesterday midday) and me noticing and doing this workaround lost?
Was there even data collected? How do I check?

Also, the post mentions something about software/OS statistics being lost/dropped. That doesn't appear to be the case for me, but can I check and make sure?

I do have daily mysql dumps available going back a month, if necessary.
